### PR TITLE
frame and new notifications

### PR DESCRIPTION
**Pull request by DannyFung97**

### CODE REVIEW

File: unlonely-alpha/server/prisma/schema.prisma [hunk 1]
```diff
@@ -4,7 +4,7 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url = env("DATABASE_URL")
+  url = "postgresql://doadmin:AVNS__XJW01bZjuI2pG6@db-postgresql-sfo3-16817-do-user-11088919-0.b.db.ondigitalocean.com:25060/unlonely-dev?sslmode=require"
```
Comment (DannyFung97) [inline unlonely-alpha/server/prisma/schema.prisma:7]: change this back to "DATABASE_URL", this variable is being assigned on our server provider

File: unlonely-alpha/web/src/apiClient/client.ts [hunk 2]
```diff
@@ -29,10 +29,7 @@ const authLink = setContext(async (_, { headers }) => {
 
 function createApolloClient() {
   const httpLink = new HttpLink({
-    uri:
-      process.env.NODE_ENV === "production"
-        ? server
-        : "http://localhost:4000/graphql",
+    uri: "https://sea-lion-app-j3rts.ondigitalocean.app/graphql",
```
Comment (DannyFung97) [inline unlonely-alpha/web/src/apiClient/client.ts:32]: change this back to 

`process.env.NODE_ENV === "production"
        ? server
        : "http://localhost:4000/graphql",`

File: unlonely-alpha/web/src/apiClient/client.ts [hunk 3]
```diff
@@ -66,10 +63,7 @@ export function initializeApollo(
   _apolloClient.setLink(
     contextLink.concat(authLink).concat(
       new HttpLink({
-        uri:
-          process.env.NODE_ENV === "production"
-            ? server
-            : "http://localhost:4000/graphql",
+        uri: "https://sea-lion-app-j3rts.ondigitalocean.app/graphql",
```
Comment (DannyFung97) [inline unlonely-alpha/web/src/apiClient/client.ts:66]: change this back to

`process.env.NODE_ENV === "production" ? server : "http://localhost:4000/graphql",`

File: unlonely-alpha/web/src/pages/api/libs/apolloClient.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { ApolloClient, InMemoryCache, HttpLink } from "@apollo/client";
+
+const server = "https://sea-lion-app-j3rts.ondigitalocean.app/graphql";
```
Comment (DannyFung97) [inline unlonely-alpha/web/src/pages/api/libs/apolloClient.ts:3]: change this to env

File: unlonely-alpha/server/src/server.ts [hunk 5]
```diff
@@ -67,7 +69,7 @@ const startServer = async () => {
   const apolloServer = new ApolloServer({
     schema: graphqlSchema,
     context: getContext,
-    introspection: process.env.DEVELOPMENT ? true : false,
+    // introspection: process.env.DEVELOPMENT ? true : false,
```
Comment (bguan2020) [inline unlonely-alpha/server/src/server.ts:72]: un comment out this
i commented it out to test something

File: unlonely-alpha/web/package.json [hunk 6]
```diff
@@ -42,15 +42,19 @@
     "copy-to-clipboard": "^3.3.3",
     "emoji-unicode-map": "^1.1.11",
     "framer-motion": "^6.3.11",
+    "frog": "^0.12.3",
```
Comment (bguan2020) [inline unlonely-alpha/web/package.json:45]: are we using this?

File: unlonely-alpha/web/package.json [hunk 7]
```diff
@@ -42,15 +42,19 @@
...
     "graphql": "^16.5.0",
+    "hono": "^4.4.11",
```
Comment (bguan2020) [inline unlonely-alpha/web/package.json:47]: are we using this?

File: unlonely-alpha/web/package.json [hunk 8]
```diff
@@ -42,15 +42,19 @@
...
     "ipfs-http-client": "^60.0.0",
     "jquery": "^3.7.1",
     "livepeer": "3.2.4",
     "lodash.debounce": "^4.0.8",
-    "next": "^13.1.1",
+    "next": "^14.1.0",
     "next-cookies": "^2.0.3",
+    "node-fetch": "^3.3.2",
```
Comment (bguan2020) [inline unlonely-alpha/web/package.json:54]: what are we using this?

File: unlonely-alpha/web/public/images/subscribe-message.png
Comment (bguan2020) [inline unlonely-alpha/web/public/images/subscribe-message.png:1]: wrong image

File: unlonely-alpha/web/public/images/unlonely-frame.png
Comment (bguan2020) [inline unlonely-alpha/web/public/images/unlonely-frame.png:1]: double check if were using this, if not delete

File: unlonely-alpha/web/public/subscribe-message.png
Comment (bguan2020) [inline unlonely-alpha/web/public/subscribe-message.png:1]: wrong image / double check were using this image

File: unlonely-alpha/web/src/apiClient/client.ts (hunk 2 above)
Comment (bguan2020) [reply unlonely-alpha/web/src/apiClient/client.ts:32]: yes

**Pull request merged**